src/UI/Implementation/Component/Popover/Popover.php

```diff
@@ -0,0 +1,115 @@
+<?php
+namespace ILIAS\UI\Implementation\Component\Popover;
+
+use \ILIAS\UI\Component;
+use ILIAS\UI\Component\Signal;
+use ILIAS\UI\Implementation\Component\ComponentHelper;
+use ILIAS\UI\Implementation\Component\SignalGeneratorInterface;
+
+class Popover implements Component\Popover\Popover {
+
+	use ComponentHelper;
+
+	const POS_LEFT = 'left';
+	const POS_BOTTOM = 'bottom';
+	const POS_TOP = 'top';
+	const POS_RIGHT = 'right';
+	const POS_AUTO = 'auto';
```

Comment:
We probably need to extend that with something like bottom-corner (think about our top navigation).